meson_options.txt

```diff
@@ -21,3 +21,6 @@ option('capi-ml-inference-actual', type: 'string', value: 'capi-ml-inference',
         description: 'backward compatible dependency name of capi-ml-inference')
 option('capi-ml-common-actual', type: 'string', value: 'capi-ml-common',
         description: 'backward compatible dependency name of capi-ml-common')
+
+# detect build os
+option('debian-build', type: 'boolean', value: false)
```

Comment:
How about we have a reduce test toloerance option and instead of debian build option?

Reply:
Tizen i586 has same problem, so...

Reply:
that will reduce the tolerance everywhere. for now, the problem seems limited to capi and ccapi unittests.

Reply:
Updated to reduce the tolerance for packaging, but CI runs at max tolerance (reduce tolerance enabled by default but CI will disable it)?